### ISSUE TITLE
Show Receipts in new tab, not as downloadable

### ISSUE THREAD

**Issue by rakshi00** [ElsaHilftDigital/main#101]

**Comment by rakshi00** [comment]
implemented "inline", not yet tested

**Comment by rakshi00** [comment]
Not working yet

**Comment by rakshi00** [comment]
@F1rst-Unicorn 
something to do with mime-type?

**Comment by F1rst-Unicorn** [comment]
mime type is reported wrong by telegram. the purchase u uploaded on May 05 23:36:18 was reported to be octet-stream. have you uploaded it as image? is it really an image?

**Comment by rakshi00** [comment]
Yes, it‘s an image and i‘m always uploding it as photo over telegram desktop

**Comment by ntruessel** [comment]
This may be fixed by stopping to store the data in the database

**Comment by rakshi00** [comment]
It's peculiar as originally, the picture was opened in the same browser window. At that time it annoyed me that the favicon was from react.
